cli/tests/integrations/test_marathon_pods.py

```diff
@@ -0,0 +1,173 @@
+
+
+def test_pod_list():
+    paths = [GOOD_POD_FILE_PATH, DOUBLE_POD_FILE_PATH, TRIPLE_POD_FILE_PATH]
+    expected_json = [file_json(path) for path in paths]
+
+    with _pod(GOOD_POD_ID, GOOD_POD_FILE_PATH), \
+            _pod(DOUBLE_POD_ID, DOUBLE_POD_FILE_PATH), \
+            _pod(TRIPLE_POD_ID, TRIPLE_POD_FILE_PATH):
+
+        exit_status, stdout = _pod_list(json=True)
+        assert exit_status == 0
+
+        parsed_stdout = json.loads(stdout.decode('utf-8'))
+        _assert_same_elements(parsed_stdout, expected_json)
+
+        exit_status, stdout = _pod_list(json=False)
+        assert exit_status == 0
+
+        assert stdout == file_bytes('cli/tests/unit/data/pod.txt')
+
+
+def test_pod_show():
+    with _pod(GOOD_POD_ID, GOOD_POD_FILE_PATH):
+        exit_status, stdout = _pod_show(GOOD_POD_ID)
+        assert exit_status == 0
+
+        good_pod_json = file_bytes(GOOD_POD_FILE_PATH)
+        assert stdout == good_pod_json
+
+
+def test_pod_update_from_file():
+    with _pod(GOOD_POD_ID, GOOD_POD_FILE_PATH):
+        exit_status, stdout = _pod_update_from_file(GOOD_POD_ID,
+                                                    UPDATED_GOOD_POD_FILE_PATH,
+                                                    force=False)
+        assert exit_status == 0
+        assert re.fullmatch('Created deployment \S+\n', stdout.decode('utf-8'))
+
+        exit_status, stdout = _pod_show(GOOD_POD_ID)
+        assert exit_status == 0
+
+        updated_good_pod_json = file_bytes(UPDATED_GOOD_POD_FILE_PATH)
+        assert stdout == updated_good_pod_json
+
+
+def test_pod_update_from_stdin_force_true():
+    with _pod(GOOD_POD_ID, GOOD_POD_FILE_PATH):
+        exit_status, stdout = \
+            _pod_update_from_stdin(GOOD_POD_ID,
+                                   UPDATED_GOOD_POD_FILE_PATH,
+                                   force=True)
+        assert exit_status == 0
+        assert re.fullmatch('Created deployment \S+\n', stdout.decode('utf-8'))
+
+        exit_status, stdout = _pod_show(GOOD_POD_ID)
+        assert exit_status == 0
+
+        updated_good_pod_json = file_bytes(UPDATED_GOOD_POD_FILE_PATH)
+        assert stdout == updated_good_pod_json
+
+
+_POD_BASE_CMD = ['dcos', 'marathon', 'pod']
+_POD_ADD_CMD = _POD_BASE_CMD + ['add']
+_POD_LIST_CMD = _POD_BASE_CMD + ['list']
+_POD_REMOVE_CMD = _POD_BASE_CMD + ['remove']
+_POD_SHOW_CMD = _POD_BASE_CMD + ['show']
+_POD_UPDATE_CMD = _POD_BASE_CMD + ['update']
+
+
+def _pod_add_from_file(file_path):
+    cmd = _POD_ADD_CMD + [file_path]
+    exit_status, stdout, stderr = exec_command(cmd)
+    return exit_status
+
+
+def _pod_add_from_stdin(file_path):
+    cmd = _POD_ADD_CMD
+    exit_status, stdout, stderr = _exec_command_with_stdin(cmd, file_path)
+    return exit_status
+
+
+def _pod_list(json):
+    cmd = _POD_LIST_CMD + (['--json'] if json else [])
+    exit_status, stdout, stderr = exec_command(cmd)
+    return exit_status, stdout
+
+
+def _pod_remove(pod_id, force):
```

Comment:
I would make s/force/args or extra_args. If the test is testing the command with more flags, they should be more explicit.